docs/07-deployments/04-deploying-to-aws.md

```diff
@@ -146,6 +181,12 @@ Chances are that if you followed the instructions, you have a Serverpod deployme
 
 You can find a list of your currently running EC2 instances by navigating to _EC2 > Instances_. Click on one of the instances to go to its summary page. From there, click _Connect_. On the _Connect to instance_ page, click _Connect_, and AWS will open up a console window with access to your EC2 instance.
 
+After Signing in to your instance, you should check if the service is running with `systemctl status serverpod.service`
+
+If the service is running, you can look into the serverpod error log in `serverpod.err` and server log in `serverpod.log` in the home directory
+
+If all checks out, try to use `curl localhost:8082` to see if the service can be reached from local. If we get the expected response, we know that the service is running properly.
```

Comment:
Suggestion: change the port to the API port as that is guaranteed to be running, the web server can potentially be turned off depending on how you have configured Serverpod.

e.g.

curl localhost:8080